pkg/virt-config/featuregate/active.go

```diff
@@ -128,6 +128,12 @@ const (
 	//
 	// PasstIPStackMigration enables seamless migration with passt network binding.
 	PasstIPStackMigration = "PasstIPStackMigration"
+
+	// Owner: @DariuszPorowski
+	// Alpha: v1.?.0
+	// This feature is disabled by default. When enabled, it allows setting specific Hyper-V
+	// fields on the VMI spec which are otherwise restricted.
+	HyperVL1VH = "HyperVL1VH"
```

Comment:
Would recommend using more general "HyperVLayered" or "HyperVLayeredEnabled" as the feature gate name